mangopay/tests/bank_account.py

```diff
@@ -4,20 +4,51 @@
 
 from ..models import MangoPayBankAccount
 
-from .factories import MangoPayBankAccountFactory
+from .factories import (MangoPayIBANBankAccountFactory,
+                        MangoPayOTHERBankAccountFactory)
 from .client import MockMangoPayApi
 
+from ..constants import (BA_BIC_IBAN,
+                         BA_OTHER)
+
 
 class MangoPayBankAccountTests(TestCase):
 
     def setUp(self):
-        self.bank_account = MangoPayBankAccountFactory()
+        self.bank_account_iban = MangoPayIBANBankAccountFactory()
+        self.bank_account_other = MangoPayOTHERBankAccountFactory()
+        self.bank_account_us = MangoPayOTHERBankAccountFactory(account_type="US")
+
+    @patch("mangopay.models.get_mangopay_api_client")
+    def test_create_iban_bank_account(self, mock_client):
+        id_ = 33322
+        mock_client.return_value = MockMangoPayApi(bank_account_id=id_)
+        self.assertIsNone(self.bank_account_iban.mangopay_id)
+        self.bank_account_iban.create()
+        MangoPayBankAccount.objects.get(id=self.bank_account_iban.id,
+                                        mangopay_id=id_)
+        self.assertEqual(self.bank_account_iban.account_type,
+                         BA_BIC_IBAN)
+        self.assertTrue(self.bank_account_iban.iban is not "")
+        self.assertTrue(self.bank_account_iban.account_number is None)
```

Comment:
Not very important but maybe you can use more specific assertions here, e.g.  `assertNotEqual("", self.bank_account_iban.iban)`
and
`assertIsNone(self.bank_account_iban.account_number)`